.github/workflows/ci.yml

```
@@ -99,4 +99,9 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: false
-          token: ${{ secrets.CODECOV_TOKEN }}
\ No newline at end of file
+          token: ${{ secrets.CODECOV_TOKEN }}
+      
+      # Test probably delete this
+      - name: Smoke Test
+        shell: bash -l {0}
+        run: python run_stac.py
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)